.vscode/settings.json

```diff
@@ -1,34 +1,37 @@
 {
-	"editor.detectIndentation": false,
-	"editor.insertSpaces": false,
-	"editor.tabSize": 4,
-	"files.exclude": {
-		"out": true,
-		"**/node_modules": true,
-		".vscode-test": true,
-		".nyc_output": true
-	},
-	"files.insertFinalNewline": true,
-	"typescript.tsdk": "node_modules/typescript/lib",
-	"editor.codeActionsOnSave": {
-		"source.organizeImports": true
-	},
-	"dart.extensionLogFile": "logs/extension.txt",
-	"dart.analyzerLogFile": "logs/analyzer.txt",
-	//"dart.analyzerInstrumentationLogFile": "logs/analyzer.instrum.txt",
-	"dart.flutterRunLogFile": "logs/flutter_run.txt",
-	"dart.flutterTestLogFile": "logs/flutter_test.txt",
-	"dart.dartTestLogFile": "logs/dart_test.txt",
-	"dart.flutterDaemonLogFile": "logs/flutter_daemon.txt",
-	"dart.vmServiceLogFile": "logs/observatory.txt",
-	"dart.dapLogFile": "logs/dap.txt",
-	"dart.analysisExcludedFolders": [
-		"src/test/test_projects/bazel_workspace",
-		"src/test/test_projects/dart_create_template",
-		"src/test/test_projects/flutter_create_basic",
-		"src/test/test_projects/flutter_create_sample"
-	],
-	"dart.doNotFormat": [
-		"**/lib/comments.dart"
-	],
+    "dart.analyzerLogFile":            "logs/analyzer.txt",
+    "dart.dapLogFile":                 "logs/dap.txt",
+    "dart.dartTestLogFile":            "logs/dart_test.txt",
+    "dart.extensionLogFile":           "logs/extension.txt",
+    "dart.flutterDaemonLogFile":       "logs/flutter_daemon.txt",
+    "dart.flutterRunLogFile":          "logs/flutter_run.txt",
+    "dart.flutterTestLogFile":         "logs/flutter_test.txt",
+    "dart.vmServiceLogFile":           "logs/observatory.txt",
+    "editor.detectIndentation":        false,
+    "editor.insertSpaces":             false,
+    "editor.tabSize":                  4,
+    "files.insertFinalNewline":        true,
+    "typescript.tsdk":                 "node_modules/typescript/lib",
+    "files.exclude": {
+        "**/node_modules":     true,
+        ".nyc_output":         true,
+        ".vscode-test":        true,
+        "out":                 true
+    },
+    "editor.codeActionsOnSave": {
+        "source.organizeImports":      true
+    },
+    "dart.analysisExcludedFolders": [
+        "src/test/test_projects/bazel_workspace",
+        "src/test/test_projects/dart_create_template",
+        "src/test/test_projects/flutter_create_basic",
+        "src/test/test_projects/flutter_create_sample"
+    ],
+    "dart.doNotFormat": [
+        "**/lib/comments.dart"
+    ],
+    "cSpell.words": [
+        "dartdoc",
+        "oldschool"
+    ]
```

Comment:
I think this file may have been included in error, it doesn't appear to contain any relevant changes.

Reply:
I used the C-Spell plugin in vscode to check the spellings and these two words raised errors, so I added them to the workspace dictionary. You can delete it if you find it useless